e2e/e2e-helper.js

```diff
@@ -107,6 +107,19 @@ export default class Helper {
     }
   }
 
+  createNewDirectory() {
+    const newDir = `${generateRandomStr()}-local`;
```

Comment:
change the -local to -dir it has meaning (local scope / remote scope / env scope)